### DISCUSSION TITLE
How to play LAN multiplayer

### DISCUSSION THREAD

**Discussion by bittcrak** [Kron4ek/minecraft-vortex-launcher#84] [Q&A]
can I not join LAN servers with a Vortex launcher's offline account? The server is hosted on an online Microsoft account.

**Comment by Kron4ek** [comment]
Yes, you can. But you'll likely need to run a dedicated server in your LAN network and set `online-mode` to false in the server settings.

**Reply by bittcrak** [reply]
What if the server is hosted from an offline account?

**Reply by Kron4ek** [reply]
You'll probably need to run a dedicated server anyway, or maybe not, i can't say for sure.
